saleor/dashboard/product/forms.py

```diff
@@ -14,14 +14,14 @@
 
 
 class ProductClassSelectorForm(forms.Form):
-
-    def __init__(self, *args, **kwargs):
-        product_classes = kwargs.pop('product_classes', [])
-        super(ProductClassSelectorForm, self).__init__(*args, **kwargs)
-        choices = [(obj.pk, obj.name) for obj in product_classes]
-        self.fields['product_cls'] = forms.ChoiceField(
-            label=pgettext_lazy('Product class form label', 'Product type'),
-            choices=choices, widget=forms.RadioSelect)
+    """
+    Form that creates list of available product classes (types)
+    in circle selection style.
```

Comment:
By "circle selection style" you mean the floating action button? That's part of the presentation layer and has nothing to do with Python code of this form. Also, this form actually doesn't create anything. I would put it simply and write that this form allows selecting a product class.

Reply:
Ok.